### PR TITLE
Docs/add doc for subcommand

### PR DESCRIPTION
**Pull request by hellojason3**
Add some description to the user command.

### CODE REVIEW

File: docs/user-cli/AddWithdrawal.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Usage
+```bash
+city-rollup-user-cli add-withdrawal --private-key <L2 private key> --user_id <user_id> --value <ammount to withdraw in sats> --nonce <a random number for enhanced security> --destination <L1 address>
+```
+
+## Example
+Withdrawal 5 DOGE from the wallet with a private key 0ed2be709b9ac1cefb7c35449c423efd22bbfe5cff7aa08f00faad772c242a34 to the address 0x7a9f3b2f8f5e3b7f1f3f0f0f0f0f0f0f0f0f0f0f
+```bash
+city-rollup-user-cli add-withdrawal --private-key 0ed2be709b9ac1cefb7c35449c423efd22bbfe5cff7aa08f00faad772c242a34 --user-id 1 --value 500000000 --nonce 1 --destination 0x7a9f3b2f8f5e3b7f1f3f0f0f0f0f0f0f0f0f0f0f
```
Comment (0x8f701) [inline docs/user-cli/AddWithdrawal.mdx]: destination should be l1 p2pkh address

File: docs/user-cli/ClaimDeposit.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Usage
+```bash
+city-rollup-user-cli claim-deposit --private-key <dogecoin l1 private key (hex, not wif)> --txid <L1 deposit transaction hash> --user_id <user_id>
+```
+
+## Example
+Claim deposit with the txid 0x7a9f3b2f8f5e3b7f1f3f0f0f0f0f0f0f0f0f0f0f and user_id 1
+```bash
+city-rollup-user-cli claim-deposit --private-key 0ed2be709b9ac1cefb7c35449c423efd22bbfe5cff7aa08f00faad772c242a34 --txid 0x7a9f3b2f8f5e3b7f1f3f0f0f0f0f0f0f0f0f0f0f --user_id 1
```
Comment (0x8f701) [inline docs/user-cli/ClaimDeposit.mdx]: txid shouldn't have 0x prefix

Comment (0x8f701) [reply docs/user-cli/ClaimDeposit.mdx]: --user_id may not work, IIRC it should be --user-id

File: docs/user-cli/GetPublicKey.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Get Public Key
+
+## Description
+Get the public key from the private key.
+
+## Usage
+```bash
+city-rollup-user-cli get-public-key --private_key <private_key>
```
Comment (0x8f701) [inline docs/user-cli/GetPublicKey.mdx]: --private_key may not work as above

File: docs/user-cli/ResisterUser.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Register User
+
+## Description
+Register a new user on the rollup chain. 
+Users should provide an L2 private key for registration. If not, a random wallet can be generated through the `random-wallet` command.
+
+## Usage
+```bash
+city-rollup-user-cli register-user --private_key <L2 private key>
```
Comment (0x8f701) [inline docs/user-cli/ResisterUser.mdx]: same as above

File: docs/user-cli/SignHash.mdx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Sign Hash
+
+## Description
+Sign the action hash with the user's private key.
+
+## Usage
+```bash
+  city-rollup-user-cli sign-hash --private_key <private_key> --action_hash <action_hash> --output <output>
```
Comment (0x8f701) [inline docs/user-cli/SignHash.mdx]: same as above for --private_key and --action_hash

File: docs/user-cli/TokenTransfer.mdx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Transfer
+
+## Description
+Transfer tokens from one user to another user on the rollup chain.
+
+## Usage
+```bash
+city-rollup-user-cli transfer --private_key <L2 private key> --value <ammount in sats> --from <sender_user_id> --to <receiver_user_id> --nonce <specific to enhanced security>
```
Comment (0x8f701) [inline docs/user-cli/TokenTransfer.mdx]: same as above

**Review by cf — APPROVED** [review]
looks good

**Pull request merged**